crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -883,16 +883,6 @@ where
                             return Some(Ok(()))
                         }
 
-                        let current_state = match self.forkchoice_state {
-                            Some(state) => state,
-                            None => {
-                                // This is only possible if the node was run with `debug.tip`
-                                // argument and without CL.
-                                warn!(target: "consensus::engine", "No forkchoice state available");
-                                return None
-                            }
-                        };
-
```

Comment:
@Rjected I changed the order of checks here

Reply:
this is a good change actually, because otherwise the continuous canon chain update wouldn't run if the forkchoice state were `None` (this is the case in both `debug.tip` and `debug.continuous` i think)